src/compile/data/nullfilter.ts

```diff
@@ -17,12 +17,9 @@ const DEFAULT_NULL_FILTERS = {
 
 /** Return Hashset of fields for null filtering (key=field, value = true). */
 function parse(model: Model): Dict<FieldDef> {
-  const filterInvalid = model.filterInvalid();
-
   return model.reduceFieldDef(function(aggregator: Dict<FieldDef>, fieldDef: FieldDef) {
-    if (fieldDef.field !== '*') { // Ignore * for count(*) fields.
-      if (filterInvalid ||
-        (filterInvalid === undefined && fieldDef.field && DEFAULT_NULL_FILTERS[fieldDef.type])) {
+    if (fieldDef.aggregate !== 'count') { // Ignore * for count(*) fields.
+      if (fieldDef.field && DEFAULT_NULL_FILTERS[fieldDef.type]) {
```

Comment:
It seems like we always filterInvalid for now.  If you wanna keep the functionality, at least add it to config to allow disabling?